lib/tasks/projects.rake

```diff
@@ -33,19 +33,27 @@ namespace :projects do
       ).data
 
       assignments.each do |assignment|
+
+        assignable_project = projects[assignment.assignable_id]
         puts " #{projects[assignment.assignable_id].tags};"
 
+        puts assignment.assignable_id
+        puts assignable_project
+
+        if assignable_project.name
+          unless assignable_project.tags.data.any? { |custom_field| custom_field.has_value?("cyber")  }
+            project = Project.find_or_initialize_by(tenk_id: assignable_project.id)
+
+            project.attributes = {
+              name: assignable_project.name,
+              starts_at: assignable_project.starts_at,
+              ends_at: assignable_project.ends_at,
+              client: assignable_project.client,
+              phase_name: assignable_project.phase_name,
+              archived: assignable_project.archived
+            }
```

Comment:
Does this combine the attributes or overwrite them? Does a project still have a `tenk_id` after this? Why this, rather than `assign_attributes` or `update`?